### DISCUSSION TITLE
OpenDJ with 1 million records takes lot of memory and time for startup

### DISCUSSION THREAD

**Discussion by nileshnikalje** [OpenIdentityPlatform/OpenDJ#177] [Q&A] [answered]
We are having openDJ installation with 1 million rows. The server needs 48GB memory allocated to heap to start without giving out of memory errors. It takes almost an hour to start and it seems its taking all the time to load the DB entries into the memory.  The db directory size is 8 GB. If the db-cache-size is set to 10 Gb, it still throws out of memory errors.

Is there any config for not loading all the entries in memory? We are using PDB backend. 

Any pointers will help in troubleshooting the issue

**Comment by vharseko** [comment, marked as answer]
try switch to JE
